cmd/skopeo/utils.go

```diff
@@ -200,6 +192,22 @@ func (opts *imageDestOptions) newSystemContext() (*types.SystemContext, error) {
 	return ctx, err
 }
 
+func (opts *globalOptions) newSystemContext() *types.SystemContext {
```

Comment:
It’s inconvenient to have the `globalOptions` definition/use split between `main.go` and `utils.go` like this. 

The flag definitions could be split from `createApp`, but maybe it would be simplest to just move this function to `main.go`, beside the existing `globalOptions.getPolicyContext` and the like — and the same for the new test.